thesis-de/content-de.tex

```diff
@@ -361,11 +361,145 @@ \subsubsection{Durchführung der Evaluierung}
+Übersichtlichkeit dienen, welche Anforderungen das Frontend haben muss). Bei 
+den Kriterien, musste darauf geachtet werden, dass diese zu Projektübergabe
+realistisch und machbar durchgeführt werden konnten. Nachdem die einzelnen
+Kriterien zusammen mit dem Projektteam erstellt worden sind, wurden diese im
+Anschluss von jedem einzelnen Projektmitglied auf die Machbarkeit und
+Durchführung geprüft. Erst hiernach wurden diese der Kriterien Liste
+aufgenommen.
+
+Die einzelnen Kriterien wurden im nachhinein der Wichtigkeit absteigend,
+wie folgt aufgelistet:
+\begin{outline}
+  \1 Es muss eine Authentisierung, Autentifizierung und Autorisierung
+  gewährleistet werden. Dies bedeutet, dass die Daten und damit auch die View
+  vor ungeschützten Zugriff dritter geschützt sein muss. Ebenfalls müssen die
+  Daten selber, welche von dem Frontend abgegriffen werden, vor dritter
+  geschützt werden. Der Zugriff auf das Frontend soll über HTTPS durchgeführt
+  werden können und je nach Kundenanforderung auch Self Signed Zertifikate 
+  zulassen. Die Anmeldung am Frontend kann entweder über einen Lokalen
+  Administrator, als auch über eine LDAP(Lightweight Directory Access Protocol)
+  Anbindung verfügen, um hier z.B. Unternehmensverzeichnisse Anbinden zu können.
+  \1 Eine gute und große Community und Dokumentation über das Webfrontend.
+  Damit ist gemeint, das der Auftraggeber und auch die Projektmitglieder selber,
+  keine große Schulung mit dem Umgang des Frontends benötigen. Dies erleichtert
+  zum einen den späteren Betrieb, als auch die Wartung im späteren Verlauf,
+  wenn das Produkt bei einem Kunden eingesetzt wird.
+  \1 Eine Leichte Anpassung, Konfiguration und direkte Ausgabe der
+  Visualisierung von CPU, RAM, Netzwerk oder ähnliche Datenwerte. Hiermit ist
+  gemeint das Gewährleistet werden muss, dass  die einzelnen Graphen schnell
+  und einfach (Endanwender spezifisch) direkt angepasst werden können.
+  Ebenso muss die Anbindung an eventuell bereits schon vorhandene
+  Datenbanksysteme einfach durchgeführt werden können.
+  \1 Die Datensicherung muss einen wesentlichen Aspekt bei dem Frontend sein.
+  Dabei muss das Frontend, bereits schon Möglichkeiten bieten, eine
+  Datensicherung des Dashboards oder ähnlichem durchführen zu können.
+  Das Dashboard, wo alle Graphen und Werte hinterlegt sind, können auf ein
+  externes System (FileServer o.a.) abgelegt werden und können somit einfach
+  dupliziert und gesichert werden.
+  \1 API Zugriff für das Frontend. Damit ist gemeint, das das Frontend die
+  Möglichkeit bietet, Datensätze direkt mittels einer bereitgestellten API
+  Abfrage an der Datenbank abgreifen zu können und visuell bereit zu stellen.
+  Dabei blablabla (hier muss nochwas hin, fällt mir gerade aber nicht ein)
+  \1 Plugin Erweiterung. Das Frontend, muss eine Plugin Erweiterung
+  gewährleisten. Es dient dazu, das im späteren Verlauf, wenn das Produkt
+  bereits beim Kunden eingesetzt wird, auf bestimmte Kundenwünsche direkt
```

Comment:
Plugins werden dann in welcher Entwicklersprache entwickelt?
Ist es wichtig das es eine gängie Sprache ist? Hat das Unternehmen bereits Entwickler in der Sprache? 
Bevorzugt das Unternehmen eine Sprache der Erweiterungen? Wie viel/weit darf Erweitert werden? Wo sind Grenzen?